DFXR3Editor/Defs/DefActionID.xml

```diff
@@ -530,37 +662,37 @@
 				<field name="" dataType="f32"/>
 			</Fields2>
 		</ActionID>
-		<ActionID ID="606" name="Trail[Texture, Animation]">
+		<ActionID ID="606" name="Trails">
```

Comment:
inconsistent use between tracer and trail, it should be consistent across the program